app/src/main/res/values-night/colors.xml

```diff
@@ -180,13 +180,13 @@
 
     <!-- Search Widget -->
     <color name="search_widget_background">@color/inset_normal_theme</color>
-    <color name="search_widget_mic_fill_color">@color/primary_text_dark_theme</color>
-    <color name="search_widget_text">@color/primary_text_dark_theme</color>
+    <color name="search_widget_mic_fill_color">@color/photonLightGrey05</color>
+    <color name="search_widget_text">@color/fx_mobile_text_color_primary</color>
 
     <!-- Reader View colors -->
-    <color name="mozac_feature_readerview_text_color">@color/primary_text_dark_theme</color>
+    <color name="mozac_feature_readerview_text_color">@color/fx_mobile_text_color_primary</color>
 
     <!-- Tab Counter colors -->
-    <color name="mozac_ui_tabcounter_default_tint">@color/primary_text_dark_theme</color>
-    <color name="mozac_ui_tabcounter_default_text">@color/primary_text_dark_theme</color>
+    <color name="mozac_ui_tabcounter_default_tint">@color/photonLightGrey05</color>
```

Comment:
I suspect we might want to use `fx_mobile_text_color_primary` here, but opting out for the time being until working with the design tokens become clearer. I can confidently say the other variables should use `fx_mobile_text_color_primary` since they are all relate to text.